governance.md

```diff
@@ -0,0 +1,174 @@
+running such elections, and running other votes of the membership as required. The Election Chair
+will be filled by a Steering Committee member in the first year of their term for a duration of 1
+year.
+
+#### Secretary
+
+The Secretary will be responsible for ensuring that notes are taken in each meeting of the Committee
+and published after the meeting. In the absence of the Secretary, the presiding officer of the
+meeting will appoint another member to take on this responsibility.
+
+### Resignations
+
+A Steering Committee member may choose to resign by informing the Steering Committee Chair via
+email.  Additionally, the Steering Committee Chair can resign by informing the full committee via
+email. The resignation will take effect at the time specified in the notice, and, unless otherwise
+specified, formal acceptance of the resignation is not necessary.
+
+### Removal
+
+Steering Committee members can be removed by a two-thirds (⅔) vote of the full Steering Committee.
+In the event that that Committee is considering the removal of the Chair, the Vice-Chair will
+preside.
+
+### Vacancies
+
+If a seat on the Steering Committee becomes vacant, the remaining members of the Steering Committee
+may choose to appoint someone to fill the position for the remainder of the term. If the Steering
+Committee chooses to fill the position, the replacement will be chosen by a majority vote of the
+remaining members of the Steering Committee.  In the event that a majority vote cannot be achieved,
+the seat shall remain open until the end of the seat's term.
+
+## Committees, Working Groups, and Community Involvement
+
+Organizational functions of the US-RSE Association will be managed by standing committees that may
+consist of one or more members of the Steering Committee and interested members of and participants in
+the Association. These organizational activities include communications (website, social media,
+etc.), conference planning, and others to be determined by the Steering Committee.
+
+Other activities, events, and efforts that have a fixed end date will be managed by working groups.
+These working groups can and should be initiated by members at large through discussion with the
+Steering Committee to ensure alignment with US-RSE priorities.  Working groups should have at least
+three (3) US-RSE members as participants.
+
+## Changes to Governance
+
+Minor revisions to this document can be made with the approval of at least two-thirds (⅔) of the current
+steering committee members. Major changes to the governance structure or this document will require
+a vote of the membership directly.
+
+## Ratification
+
+This document was ratified by the following members of the First US-RSE Steering Committee on June
+XX, 2020:
+
+* Jeffrey C. Carver
+* Ian Cosden (chair)
+* Charles Ferenbaugh
+* Sandra Gesing (vice-chair)
+* Chris Hill
+* Daniel S. Katz
+* Christina Maimone (election chair)
+* Lance Parsons
+* Jordan Perr-Sauer (secretary)
```

Comment:
In addition to the author list, I think having such a "signatures" section may become cumbersome over time. What happens if a change is made down the road after one or more of us are no longer participating?

As this isn't a declaration of independence, I'm not sure what the value is of having our "signatures" here. Thoughts?

Reply:
it's fine with me to remove it

Reply:
I liked the signature section before this moved to github. The approvers will be clear so I think it's ok to remove it, but it's possible there is a good reason to have it in there. We should get agreement from everyone (including those who have already approved).

Reply:
I'm fine with either keeping it or removing it. It's probably simpler to remove.

Reply:
If we do remove it, we will need to add the positions that people have taken, (chair, vice-chair, election chair, and secretary) via the transition letter, and imho, a page on the website (perhaps with short bios).

Reply:
There is already a page on the website. I suggest we simply keep the page up to date with the results of each election. It would also be nice if we keep a list of "former members of the steering committee" on that same web page - or easily accessible from that page.

https://us-rse.org/steering-committee/

Reply:
agreed

Reply:
Sounds good. Agree that we should put the roles on the webpage. I added chair months ago before we had the others.